lib/helpers/eventListeners.js

```diff
@@ -57,16 +61,25 @@ export const addMouseEventListener = (params, { textEditor, eventListenerManager
     showDelay,
   } = preferences;
 
-  if (!showTooltipWithMouseEvent || !view) {
-    return;
-  }
-
   // No need ro register it again.
   const currentEventListener = eventListenerManager.get(textEditor);
   if (currentEventListener) {
     return;
   }
 
+  const clickEventListener = onMouseClick({ view, atom, store })
```

Comment:
Here as well, you're only using `view` no need to pass more values 😄

Reply:
Indeed, it was a copypaste error 😅